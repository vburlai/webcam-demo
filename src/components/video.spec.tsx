import { h } from 'preact'
import Video from './video'

describe('Video component', () => {
    it('should render', () => {
        expect(<Video />).toMatchSnapshot()
    });

    it('should pass class prop', () => {
        expect(<Video class="test" />).toMatchSnapshot()
    })
})