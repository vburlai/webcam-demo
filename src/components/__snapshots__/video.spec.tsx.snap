// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video component should pass class prop 1`] = `
VNode {
  "attributes": Object {
    "class": "test",
  },
  "children": Array [],
  "key": undefined,
  "nodeName": [Function],
}
`;

exports[`Video component should render 1`] = `
VNode {
  "attributes": undefined,
  "children": Array [],
  "key": undefined,
  "nodeName": [Function],
}
`;
